config/betterquesting/DefaultQuests/Quests/RewardArea-AAAAAAAAAAAAAAAAAAAAKg==/HauteCuisineDess-AAAAAAAAAAAAAAAAAAACqA==.json

```diff
@@ -31,7 +31,7 @@
       "lockedProgress:1": 0,
       "autoClaim:1": 0,
       "isSilent:1": 0,
-      "desc:8": "Trade three Haute Cuisine lootbags to get a better one: Dessert.\n\n§3BTW, there\u0027s only the marshmallow in this one, so it\u0027s not really worth it."
+      "desc:8": "Trade six Haute Cuisine lootbags to get a better one: Dessert.\n\n§3BTW, there\u0027s only the marshmallow in this one, so it\u0027s not really worth it."
```

Comment:
you sure they are six ?

Reply:
ah, seems it was fixed, I wrote these out while playing on 2.4.0 with a few people and forgot to verify :p

Reply:
nvm, read the wrong quest, is indeed six, looked at haute making quest not dessert one... adhd
![image](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/assets/38961580/15261434-ad17-4c0a-847d-e356af7ad09d)

Reply:
https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/blob/master/config/betterquesting/DefaultQuests/Quests/RewardArea-AAAAAAAAAAAAAAAAAAAAKg%3D%3D/HauteCuisineDess-AAAAAAAAAAAAAAAAAAACqA%3D%3D.json#L48

Reply:
Requires 6 items